troubleshooting/image_projection_not_consistent_across_nodes.md

```diff
@@ -0,0 +1,68 @@
+# Image projection inconsistent across nodes
```

Comment:
Please rename both the markdown file and table-of-content entry in `troubleshooting/README.md` to match this header. "inconsistent" sounds better than "not consistent."

Reply:
Done.